Documentation/libre-fix-bt-connection-issues.md

```diff
@@ -1,15 +1,11 @@
-
-
 # If you have a lot of missed readings:
-
 You can try other settings:
-
```

Comment:
Empty lines after headings are not required according to the spec, but improving the readability in plain text mode. Please add/keep it.